website/docs/build-apps/how-to-guides/display-search-and-filter-list-data.md

```diff
@@ -0,0 +1,67 @@
+---
+description: This page shows you how to display and lookup data in a List widget. 
+---
+
+# Display and Lookup Data in List
+
+<div style={{ position: "relative", paddingBottom: "calc(50.520833333333336% + 41px)", height: "0", width: "100%" }}>
```

Comment:
In arcade you are dropping a container with a list and a text widget. and that looks confusing to users. in the steps, the search part is shown towards the end. also we can skip the search and filter pieces in the arcade. Focus mainly on the display list items

Reply:
This arcade serves as an important piece when users want to implement search. Shall we move the arcade to the search section?